### PR TITLE
Automatically bold data from search in user-select template

### PR DESCRIPTION
**Pull request by skjnldsv**
Protected against xss 
Capture d’écran_2019-08-02_17-04-42

### CODE REVIEW

File: src/components/Multiselect/AvatarSelectOption.vue [hunk 1]
```diff
@@ -130,12 +129,26 @@ export default {
...
 	},
 
 	methods: {
-		toKebabCase: (string) => {
+		toKebabCase(string) {
 			return string.replace(/([a-z0-9]|(?=[A-Z]))([A-Z])/g, '$1-$2').toLowerCase()
+		},
+		highlightPhrase(text) {
+			if (!this.search.length) return text
+			return text.replace(this.search, `<strong>${this.search}</strong>`)
```
Comment (juliushaertl) [inline src/components/Multiselect/AvatarSelectOption.vue]: This only works for the first occurrence

Comment (skjnldsv) [reply src/components/Multiselect/AvatarSelectOption.vue]: how so?

Comment (juliushaertl) [reply src/components/Multiselect/AvatarSelectOption.vue]: > If pattern is a string, only the first occurrence will be replaced.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

Comment (skjnldsv) [reply src/components/Multiselect/AvatarSelectOption.vue]: Ah yes! sure :)

Comment (skjnldsv) [reply src/components/Multiselect/AvatarSelectOption.vue]: Capture d’écran_2019-08-02_17-17-43
Fixed

**Pull request merged**